lib/helper/Playwright.js

```diff
@@ -1675,19 +1675,9 @@ class Playwright extends Helper {
   async waitForEnabled(locator, sec) {
     const waitTimeout = sec ? sec * 1000 : this.options.waitForTimeout;
     locator = new Locator(locator, 'css');
-    const matcher = await this.context;
-    let waiter;
     const context = await this._getContext();
-    if (!locator.isXPath()) {
-      // playwright combined selectors
-      waiter = context.waitForSelector(`${locator.isCustom() ? `${locator.type}=${locator.value}` : locator.simplify()} >> __disabled=false`, { timeout: waitTimeout });
-    } else {
-      const enabledFn = function ([locator, $XPath]) {
-        eval($XPath); // eslint-disable-line no-eval
-        return $XPath(null, locator).filter(el => !el.disabled).length > 0;
-      };
-      waiter = context.waitForFunction(enabledFn, [locator.value, $XPath.toString()], { timeout: waitTimeout });
-    }
+    // playwright combined selectors
```

Comment:
Simplified the code path by just using waitForSelector, no need for a special xpath case